README.md

```
@@ -1 +1,116 @@
+    FastAPI->>Database: Create new case
+    Database-->>FastAPI: Case created
+    FastAPI-->>Client: Return case details
+
+    Client->>FastAPI: GET /cases/{case_id}
+    FastAPI->>Database: Retrieve case by ID
+    Database-->>FastAPI: Return case details
+    FastAPI-->>Client: Send case details
+
+## Proposed Architecture
+
+```mermaid
```

Markdownlint (0.35.0)

Expected: indented; Actual: fenced
Code block style

(MD046, code-block-style)